tools/cmake/app.cmake

```diff
@@ -9,17 +9,13 @@ endif()
 
 include(${MBED_CONFIG_PATH}/mbed_config.cmake)
 
-# Set default toolchain file
+# Load toolchain file
 if(NOT CMAKE_TOOLCHAIN_FILE OR MBED_TOOLCHAIN_FILE_USED)
     set(MBED_TOOLCHAIN_FILE_USED TRUE CACHE INTERNAL "")
-
-    set(CMAKE_TOOLCHAIN_FILE "${MBED_PATH}/tools/cmake/toolchain.cmake" CACHE INTERNAL "")
```

Comment:
`CMAKE_TOOLCHAIN_FILE ` this should stay being set if it was not set by an app

Reply:
Nope, we no longer need to use the CMAKE_TOOLCHAIN_FILE variable at all!  Including the toolchain file with include() removes the need for it.